cmd/nodecmd/create.go

```diff
@@ -485,6 +516,24 @@ func createNodes(_ *cobra.Command, args []string) error {
 	return nil
 }
 
+func promptSetUpMonitoring() (bool, bool, error) {
+	var err error
+	if !separateMonitoringInstance && existingMonitoringInstance == "" {
```

Comment:
can we add a flag to node create and node devnet wiz commands so prompting can be completelly avoided if the user knows what he wants to do beforehand (eg case 1 do not setup monitoring, or case 2 do setup monitoring with separate instance or not)